exonum/src/blockchain/config.rs

```diff
@@ -74,6 +74,28 @@ pub struct ConsensusConfig {
 impl ConsensusConfig {
     /// Default value for max_message_len.
     pub const DEFAULT_MAX_MESSAGE_LEN: u32 = 1024 * 1024; // 1 MB
+
+    /// Checks if propose timeout is less than round timeout. Warns if fails.
+    pub fn check_timeout_recommendations(&self) {
+        let propose_timeout = match self.timeout_adjuster {
+            TimeoutAdjusterConfig::Constant { timeout } => timeout,
+            TimeoutAdjusterConfig::Dynamic { max, .. } => {
+                max
+            }
+            TimeoutAdjusterConfig::MovingAverage { max, .. } => {
+                max
+            }
+        };
+
+        if self.round_timeout <= 2 * propose_timeout {
+            warn!(
+                "It is recommended that round_timeout({}) be at least twice as large \
```

Comment:
`be` -> `is` or `should be`

Also I think you should explain numbers in parens, like "round_timeout (current: {})" and "propose_timeout (current: {})"

Reply:
Thanks for pointing that out. The problem with adding "current: " is that timeouts in the message are not necessarily current. There are 2 possible scenarios when the warning is called: 
1. Invalid timeouts are in genesis block (in this case they are indeed current);
2. Invalid timeouts are specified in a new propose (in this case they are not yet current (until propose is accepted by supermajority)).